server/Heartbeat.js

```diff
@@ -0,0 +1,15 @@
+class Heartbeat {
+    constructor(systemId,flicLastSeenTime, flicLastPingTime, heartbeatLastSeenTime, systemName, hidden, sentAlerts, muted, twilioAlertNumber, heartbeatAlertRecipients) {
+        this.systemId = systemId
+        this.flicLastSeenTime = flicLastSeenTime
+        this.flicLastPingTime = flicLastPingTime
+        this.heartbeatLastSeenTime = heartbeatLastSeenTime
+        this.systemName = systemName
+        this.hidden = hidden
+        this.sentAlerts = sentAlerts
+        this.muted = muted
+        this.twilioAlertNumber = twilioAlertNumber
+        this.heartbeatAlertRecipients = heartbeatAlertRecipients
```

Comment:
yeah it could. Right now our practices assume that everyones getting all of the alerts so I could go either way on this tbh

Reply:
I like that this is in the DB now, but I'm not a huge fan of calling it `Heartbeat`. Is it really a single heartbeat's worth of data? No, what it actually represents is a Brave Hub. So maybe a better name would be `BraveHub` or something similar.

Reply:
I actually am not sure about this - while heartbeat can refer to a single pulse it can just as easily refer to the steady, recurrent pattern. The information in the table is about Brave hubs, but only in so far it relates to verifying, displaying, and alerting about their activity or their "heartbeat" - none of the other information about BraveHubs, like the installation that they belong to or the buttons that they correspond to, is contained here. 

I don't insist either way.

Reply:
Yeah, it is funny that the hubs don't need to connect to the installations at all. And I see what you mean about the heartbeat also being the pattern. But I think the properties like `muted`, `hidden`, and `system_name` are about the hub.

I'd change it both here and in the DB, but it wouldn't be make-or-break for me.

Reply:
sounds good